internal/controllers/machine/machine_controller_status.go

```diff
@@ -650,16 +654,18 @@ func calculateDeletingConditionForSummary(machine *clusterv1.Machine) v1beta2con
 	}
 }
 
-func setAvailableCondition(_ context.Context, machine *clusterv1.Machine) {
+func setAvailableCondition(ctx context.Context, machine *clusterv1.Machine) {
+	log := ctrl.LoggerFrom(ctx)
 	readyCondition := v1beta2conditions.Get(machine, clusterv1.MachineReadyV1Beta2Condition)
 
 	if readyCondition == nil {
 		// NOTE: this should never happen given that setReadyCondition is called before this method and
 		// it always add a ready condition.
+		log.Error(errors.New("ready condition must be set before setting the available condition"), "")
```

Comment:
```suggestion
		log.Error(errors.New("Ready condition must be set before setting the Available condition"), "Failed to set Available condition")
```
(I think upper case is fine if the word itself is "spelled" upper case)